test/scenarios.py

```diff
@@ -2664,6 +2664,204 @@
+                            "/tmp/testfile-bulk-01",
+                            "/tmp/testfile-bulk-02",
+                            "/tmp/testfile-bulk-03",
+                            "/tmp/testfile-bulk-04",
+                            "/tmp/testfile-bulk-05",
+                            "/tmp/testfile-bulk-06",
+                            "/tmp/testfile-bulk-07",
+                            "/tmp/testfile-bulk-08",
+                            "/tmp/testfile-bulk-09",
+                            "/tmp/testfile-bulk-10",
+                        ],
+                    ),
+                    # fmt: off
+                    action.Wait(
+                        event.Queued(0, "DROP_PEER_STIMPY", [
+                            norddrop.QueuedFile(FILES["testfile-bulk-01"].id, "testfile-bulk-01", 10485760, "/tmp"),
+                            norddrop.QueuedFile(FILES["testfile-bulk-02"].id, "testfile-bulk-02", 10485760, "/tmp"),
+                            norddrop.QueuedFile(FILES["testfile-bulk-03"].id, "testfile-bulk-03", 10485760, "/tmp"),
+                            norddrop.QueuedFile(FILES["testfile-bulk-04"].id, "testfile-bulk-04", 10485760, "/tmp"),
+                            norddrop.QueuedFile(FILES["testfile-bulk-05"].id, "testfile-bulk-05", 10485760, "/tmp"),
+                            norddrop.QueuedFile(FILES["testfile-bulk-06"].id, "testfile-bulk-06", 10485760, "/tmp"),
+                            norddrop.QueuedFile(FILES["testfile-bulk-07"].id, "testfile-bulk-07", 10485760, "/tmp"),
+                            norddrop.QueuedFile(FILES["testfile-bulk-08"].id, "testfile-bulk-08", 10485760, "/tmp"),
+                            norddrop.QueuedFile(FILES["testfile-bulk-09"].id, "testfile-bulk-09", 10485760, "/tmp"),
+                            norddrop.QueuedFile(FILES["testfile-bulk-10"].id, "testfile-bulk-10", 10485760, "/tmp"),
+                        ]),
+                    ),
+                    # fmt: on
+                    # fmt: off
+                    # Wait for some of the files and the stop
+                    action.Repeated([action.WaitForOneOf([
+                        event.Start(0, FILES["testfile-bulk-01"].id),
+                        event.Start(0, FILES["testfile-bulk-02"].id),
+                        event.Start(0, FILES["testfile-bulk-03"].id),
+                        event.Start(0, FILES["testfile-bulk-04"].id),
+                        event.Start(0, FILES["testfile-bulk-05"].id),
+                        event.Start(0, FILES["testfile-bulk-06"].id),
+                        event.Start(0, FILES["testfile-bulk-07"].id),
+                        event.Start(0, FILES["testfile-bulk-08"].id),
+                        event.Start(0, FILES["testfile-bulk-09"].id),
+                        event.Start(0, FILES["testfile-bulk-10"].id),
+                    ])], 4),
+                    # fmt: on
+                    action.Stop(),
+                    # fmt: off
```

Comment:
are these #fmt comments all necessary?

Reply:
Um, they are not but the effect of the formatted on those large arrays of events is really destructive and it's hard to read